src/init/register_configure_agent.sh

```diff
@@ -266,9 +268,31 @@ main () {
 
     get_deprecated_vars
 
+    if [ -z "${WAZUH_MANAGER}" ] && [ -n WAZUH_PROTOCOL ]; then
```

Comment:
```suggestion
    if [ -z "${WAZUH_MANAGER}" ] && [ -n "${WAZUH_PROTOCOL}" ]; then
```

Don't you think?

Reply:
Done [b04ac96](https://github.com/wazuh/wazuh/pull/15038/commits/b04ac96d39822d256022cdf5fde42d57f5e043b9)